packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -387,7 +390,7 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
       forceUpdate();
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
```suggestion
```
 you can remove this now

Reply:
We intentionally avoid re-running this after virtualizerStartIndex changes so we still require non-exhaustive deps I believe.